architecture/device_tree.rst

```diff
@@ -23,34 +23,61 @@ shared across the boot stages.
     - Boot loader stages may load a device tree structure in memory for all boot
       stage to get platform configuration from. If such device tree data are to
       be accessed by the non-secure world, they shall be located in non-secure
-      memory.
+      memory. Secure world may used its content during the OP-TEE core
```

Comment:
s/used/use/

@d3zd3z is it right or wrong to use definite article (`the`) when talking about OP-TEE core? For me "the core" sounds correct, but in "the OP-TEE core" it sounds like the `the` is not needed.